python_code_review/object_orienting_programming.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+import pytz
+
+
+class Account:
+
+	@staticmethod
+	#creates a static method, meaning u don't need the self parameter anymore and this static method can be called using Account._current_time
```

Comment:
Put this comment above the `@staticmethod`

Reply:
Got it.